tests/proto/3rdparty/YBuild

```diff
@@ -17,3 +17,10 @@ CustomInstaller(
     script='install-protobuf.sh',
     deps=':build-essentials-pack'
 )
+
+PythonPackage(
+    'py-protobuf',
+    package='protobuf',
```

Comment:
Isnt it different than the proto version we use in search stack  (not sure if it matters)

Reply:
doesn't matter - it's for YBT's own test